docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
@jfrazelle IMO it would be nice if the UX for "unlimited" was something like:

```
% docker run --pids-limit=unlimited -it ubuntu bash
```
or maybe
```
% docker run --pids-limit=max -it ubuntu bash
```